docs/source/examples/workspaces/libvirt/PinFile

```diff
@@ -282,3 +285,70 @@ libvirt-network-dns:
                 - testhost
                 - testhostalias
 
+libvirt-multi-node-net:
```

Comment:
This target won't generate an inventory since there's no layout

Reply:
Why would I need a layout?